Sources/REventLogger/REvent.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+struct REvent: Codable {
+    let eventType: EventType
+    let appId: String?
+    let appName: String?
+    let appVer: String?
+    let osVer: String
+    let deviceModel: String
+    let deviceBrand: String
+    let deviceName: String
+    let sourceName: String
+    let sourceVer: String
+    let errorCode: String
+    let errorMsg: String
+    var eventVer: String = "1"
+    var platform: String = "iOS"
+    var occurrenceCount: Int = 0
+    var firstOccurrenceOn: Double // unix time
```

Comment:
nit: could be renamed to `firstOccurrenceTimestamp`.

Reply:
@anish-munirathinam @Esakkiraja-Pothikannan I prefer to have the unit specified to know just by looking at the request. e.g. `firstOccurrenceMillis` or `firstOccurrenceMs`.`Timestamp` can still be a bit ambiguous. What do you think?

Reply:
As long as both platforms are consistent it is fine. Let's stick with `firstOccurrenceOn` then.